ako-clean/cleanup.go

```diff
@@ -104,8 +101,18 @@ func (cfg *AKOCleanupConfig) Cleanup(ctx context.Context) error {
 		cleanupL4PolicySets,
 		cleanupPoolGroups,
 		cleanupPools,
-		func() error { return avirest.DeleteServiceEngines() },
-		avirest.DeleteServiceEngineGroup,
+		func() error {
+			if VPCMode {
```

Comment:
why are we not deleting ServiceEngine or SEG?

Reply:
In VPC based deployments, we have moved away from SE group per Supervisor cluster. Instead, we will be using the Default SE Group per Tenant, and this SE group can be shared multiple Supervisors mapped to the same Avi controller, so not deleting the SEs and SE Group.